### PR TITLE
[URGENT] Fix/care charges assessed

### PR DESCRIPTION
**Pull request by mugnivenko**
https://app.clickup.com/t/1vrgpkb

New condition https://app.clickup.com/t/1y20tum

### CODE REVIEW

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
blows up when:

- make new package
- add care charge with collect type 'supplier'
- ensure all dates on ongoing
notice how error appears

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

File: pages/care-package/[guid]/broker-care-charges.js [hunk 1]
```diff
@@ -43,67 +46,115 @@ export const getServerSideProps = withSession(async ({ req }) => {
...
+    claimReason,
+    claimCollector,
+    description,
+    assessmentFileName,
+    assessmentFileId,
+    endDate,
+    subType,
+    startDate,
+    hasAssessmentBeenCarried
+  } = careCharge;
```
[the remark this answers is not in the document]
Comment (mugnivenko) [reply pages/care-package/[guid]/broker-care-charges.js:90]: it is useful for
image
and more

[the remark this answers is not in the document]
Comment (akeamwoodsND) [reply pages/care-package/[guid]/broker-care-charges.js:90]: that was all achieved before without destructuring.

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
you can actually use the 'reset' method from react-hook-form to remove all those 'setValues' we were doing
Screenshot 2021-12-09 at 10 53 02

File: pages/care-package/[guid]/broker-care-charges.js [hunk 2]
```diff
@@ -43,67 +46,104 @@ export const getServerSideProps = withSession(async ({ req }) => {
...
+      startDate: startDate && new Date(startDate),
+      endDate: endDate && new Date(startDate),
+      subType,
+      careChargeCost,
+      claimReason,
+      claimCollector,
+      description,
+      assessmentFileName,
+      assessmentFileId,
+      isOngoing: endDate ? false : startDate,
```
Comment (akeamwoodsND) [inline pages/care-package/[guid]/broker-care-charges.js:135]: isOngoing: !endDate

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
care-package/ce81a80a-8f2c-4581-acc9-633b4b036e4e/broker-care-charges 
(development) this has care charges but doesnt show announcement.

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
this package has care charges but announcement/banner isnt show
care-package/9e3e135c-93cd-46a1-b819-fa83f2f5caf3/broker-care-charges

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Screenshot 2021-12-20 at 15 18 38
should provide link to care charge instead like so (also change word 'done' to 'complete'):
Screenshot 2021-12-20 at 15 21 39

**Pull request merged**